src/Operation/Factory/SubresourceOperationFactory.php

```diff
@@ -79,18 +79,13 @@ private function computeSubresourceOperations(string $resourceClass, array &$tre
             if (null === $parentOperation) {
                 $visiting = "$rootResourceClass-$property-$subresourceClass";
             } else {
-                $prefix = '';
-                $visiting = "{$parentOperation['property']}-{$parentOperation['resource_class']}-$property-$subresourceClass";
+                $suffix = "$property-$subresourceClass";
 
-                foreach ($parentOperation['identifiers'] as $key => list($param, $class)) {
-                    $prefix .= 0 === $key ? $class : "-$param-$class";
-                }
-
-                if (false !== strpos($prefix, $visiting)) {
+                if (false !== strpos($parentVisiting, $suffix)) {
```

Comment:
We're still doing `strpos`? Is there any downside to the algorithm I've described? It's much easier to understand, at least...